src/components/AboutUs/team.json

```diff
@@ -0,0 +1,345 @@
+    "title": "Lead Engineer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/in/kutay-ka%C4%9Fan-%C3%B6zen/",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/Iseluin",
+        "icon": "github"
+      }
+    ]
+  },
+  {
+    "id": "2",
+    "name": "Maher",
+    "img": "https://avatars.githubusercontent.com/u/18458259?v=4",
+    "title": "Project supervisor",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/in/maher-al-aqil-56895894/",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/maqalaqil",
+        "icon": "github"
+      }
+    ]
+  },
+  {
+    "id": "3",
+    "name": "Karzan Luqman",
+    "img": "https://avatars.githubusercontent.com/u/77196732?v=4",
+    "title": "Team Member/Fron-End Developer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/in/karzan7899",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/karzan7899",
+        "icon": "github"
+      }
+    ]
+  },
+  {
+    "id": "4",
+    "name": "Dania Diar",
+    "img": "https://ca.slack-edge.com/T03K5TU7K0E-U03NEE0RFT3-3c68f27559f8-512",
+    "title": "Web Developer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/in/dania-deyar-097558243/",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/dania-diar",
+        "icon": "github"
+      }
+    ]
+  },
+ 
+  {
+    "id": "5",
+    "name": "Rondik Mousa",
+    "img": "https://ca.slack-edge.com/T03K5TU7K0E-U03ME7K3Q07-7c77b4b5ec30-512",
+    "title": "Web Developer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/in/rondik-mousa-646653256/",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/rondikmousa",
+        "icon": "github"
+      }
+    ]
+  },
+  {
+    "id": "6",
+    "name": "Helin Faidhan",
+    "img": "https://ca.slack-edge.com/T03K5TU7K0E-U03NTLZN3MW-31f24e5340ee-512",
+    "title": "Web Developer",
+    "desc": "",
+    "links": [
+      {
+        "name": "linkedIn",
+        "url": "https://www.linkedin.com/in/helin-faidhan-a516a71aa/",
+        "icon": "linkedIn"
+      },
+      {
+        "name": "GitHub",
+        "url": "https://github.com/Helin-Faidhan",
+        "icon": "github"
+      }
+    ]
+  },
+  {
+    "id": "7",
+    "name": "Marwa Hakmi",
+    "img": "https://ca.slack-edge.com/T03K5TU7K0E-U03M7GU31T9-7f79896abdee-512",
```

Comment:
Since this profile image appears in the circle, it's breaking the design. Could you change the link to 
[slack profile image](https://ca.slack-edge.com/T03K5TU7K0E-U03M7GU31T9-c3416698ca47-512)